libfreerdp/core/test/TestFuzzCryptoCertificateDataSetPEM.c

```diff
@@ -1,23 +1,22 @@
 #include <stddef.h>
 #include <stdint.h>
-#include <freerdp/crypto/certificate.h>
+#include <freerdp/crypto/certificate_store.h>
 
 int LLVMFuzzerTestOneInput(const uint8_t* Data, size_t Size)
 {
-	char* buf = calloc(Size + 1, sizeof(char));
-	rdpCertificateData* cert_data = certificate_data_new("somehost", 1234);
-	if (!buf || !cert_data)
+	char *pem = calloc(Size + 1, sizeof(char));
+	if (!pem)
 		goto cleanup;
+	pem[Size] = '\0';
 
-	memcpy(buf, Data, Size);
-	if (!certificate_data_set_pem(cert_data, buf))
-	{
+	rdpCertificateData* data = NULL;
+	data = freerdp_certificate_data_new_from_pem("somehost", 1234, pem, strlen(pem));
```

Comment:
in this case better use `strnlen` as you know the maximum size of the string

Reply:
right, fixed